cmd/xl-storage-disk-id-check.go

```diff
@@ -821,6 +825,61 @@ func (p *xlStorageDiskIDCheck) monitorDiskStatus() {
 	}
 }
 
+// monitorDiskStatus should be called once when a drive has been marked offline.
+// Once the disk has been deemed ok, it will return to online status.
+func (p *xlStorageDiskIDCheck) monitorDiskWritable() {
+	// We check every 2 minutes if the disk is writable and we can read back.
+	t := time.NewTicker(2 * time.Minute)
+	defer t.Stop()
+	fn := mustGetUUID()
+
+	// Be just above directio size.
+	toWrite := []byte{xioutil.DirectioAlignSize + 1: 42}
```

Comment:
do we not need some data to write, are we just writing zeroes?

Reply:
Well, mostly 0s and a 42 at the end. Shouldn't really make much of a difference.